### ISSUE TITLE
Quesions on accuracy gain of WS?

### ISSUE THREAD

**Issue by huangh12** [joe-siyuan-qiao/WeightStandardization#4]
Thanks for the impressive work, but I have some questions:

Is your pretrained model necessary for WS to obatin improvement in object detection, segementation and video recognition? Your pretrained model is trained with WS in ImageNet and already benefits from the WS normalization. Therefore, is it possible the gains in object detection, segmentation may come from the gains in backbone?  This seems to has been proved in https://github.com/joe-siyuan-qiao/WeightStandardization/issues/3, where your pretrained model (trained with WS) is  not used, and the AP of gn+ws is worse than gn, in object detection.  Could you please explain it?  

Besides, the table4 shows that the GN with ResNet50/101 are 39.86 and 39.68, where ResNet101 is even lower than ResNet50. I think it's beyond understanding. Actually, in the original paper of GN, they are 40.3 and 41.8 respectively. 
![图片](https://user-images.githubusercontent.com/18748036/55713513-67013b80-5a23-11e9-8735-bf38f4a033a2.png)

Thank you!

**Comment by joe-siyuan-qiao** [comment]
Thanks for your interests in our work.

Yes, they are necessary. The training procedure of the tasks you mentioned is basically fine-tuning pre-trained models. Without using the right models pre-trained on ImageNet, in the most extreme cases, the models are basically trained from scratch (or even worse because you might give the model a bad initialization due to the model differences). Training from scratch will hurt the performance a lot because in fine-tuning tasks the training time is not long enough and/or the training data is not big enough. #3 does not support your argument because GN is using its own pre-trained models while GN+WS is not using the right one. Loading GN models with BN pre-trained parameters might be a fairer comparison, although the results are not really meaningful.

If you take a look at Table 1, you will notice that the performances of RN101 are also similar to RN50 on ImageNet. All the performances we reported are based on our implementations shown in the repo. For why this is happening, deeper networks are hard to optimize, thus normalization techniques become more important. An example is RN1202 is worse than RN101. That's might be one of the reasons why RN50 and RN101 perform similarly when they are trained with GN. But as the original implementation can achieve better results on RN101 for coco, we suspect that this might be also due to the minor implementation differences between different platforms and frameworks or the training is highly sensitive to hyper-parameters for which we might not choose the best configurations. We are actively contacting the authors of GN to see how to improve GN's performances on our implementation. After all, an improved GN is also beneficial to our method GN+WS.

Thanks.

**Comment by huangh12** [comment]
Thanks for your patient answer. It's very helpful.
